CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.0
+ - New year rollover should be handled better now when a year is not present in
```

Comment:
Maybe first line could be
* no longer memorize current year at startup, which caused incorrect timestamps after new year